docs/ref/models/expressions.txt

```diff
@@ -425,6 +425,33 @@ Conditional expressions allow you to use :keyword:`if` ... :keyword:`elif` ...
 :keyword:`else` logic in queries. Django natively supports SQL ``CASE``
 expressions. For more details see :doc:`conditional-expressions`.
 
+SubQuery expressions
+--------------------
+
+.. currentmodule:: django.db.models.expressions
```

Comment:
Is there a reason not to import them in django.db.models like the other expressions in this doc?